ci-operator/step-registry/telco5g/cnf/tests/telco5g-cnf-tests-commands.sh

```diff
@@ -351,10 +351,11 @@ elif [[ "$T5CI_JOB_TYPE" == "hcp-cnftests" ]]; then
     export HYPERSHIFT_ENVIRONMENT=true
     export FEATURES_ENVIRONMENT=hypershift-ci
 else
-    export FEATURES="${FEATURES:-sriov performance sctp xt_u32 ovn metallb multinetworkpolicy vrf bondcni tuningcni}"
+    export FEATURES="${FEATURES:-sriov performance sctp xt_u32 ovn metallb multinetworkpolicy vrf bondcni tuningcni knmstate}"
 fi
 export VALIDATIONS_FEATURES="${VALIDATIONS_FEATURES:-$FEATURES}"
 export TEST_RUN_FEATURES="${TEST_RUN_FEATURES:-$FEATURES}"
+export TEST_RUN_FEATURES="knmstate"
```

Comment:
should you remove this one?

Reply:
> should you remove this one?

Yes, I'm waiting for 
- https://github.com/openshift-kni/cnf-features-deploy/pull/2134

to do some quicker run. 

/hold